src/test/java/com/marklogic/appdeployer/command/viewschemas/ManageViewSchemasTest.java

```diff
@@ -56,8 +57,9 @@ protected void verifyResourcesWereDeleted(ResourceManager mgr) {
 	 */
 	@Override
 	protected void afterResourcesCreated() {
-		RestTemplate clientTemplate = RestTemplateUtil.newRestTemplate(appConfig.getHost(), appConfig.getRestPort(),
+		RestConfig restConfig = new RestConfig(appConfig.getHost(), appConfig.getRestPort(),
```

Comment:
Just converted this test over to use the non-deprecated RestTemplateUtil method.

Reply:
Should we keep the original test in place as well, until the deprecated function is actually deleted?

Reply:
That's a good point, I think I can alter the test to use the deprecated factory method on RestTemplateUtil.

Reply:
Test now verifies both the deprecated and non-deprecated method